index_documents.go

```diff
@@ -15,6 +15,8 @@ import (
 	"strings"
 )
 
+var NoLimit int64 = math.MinInt64
```

Comment:
We should provide comment (as documentation for users) to what this variable means, and when to use it.

Reply:
check it out

Reply:
I'll give a suggestion